jobs/grafana/templates/credentials

```diff
@@ -0,0 +1,4 @@
+[default]
```

Comment:
Looks like this template isn't referenced by the spec. It should be put under `home/.aws/...` right?